telosnetwork/open-block-explorer
# 393 bug data not persisting on table paginationnavigation

# Fixes #393 

## Test scenarios
- load explorer and wait 5 seconds to see transaction table data automatically refresh
- navigate to 'next' page or greater
- wait 10 seconds to see current data persist (does not refresh automatically)
- navigate back to the first page
- wait 5 seconds, should resume auto-refresh


## Checklist:
